test/simulator/timestamp/simulator_interface/simulator_interface.cpp

```diff
@@ -285,8 +285,44 @@ interface_commit_transaction(session_simulator *session)
 }
 
 void
-interface_prepare_transaction()
+interface_prepare_transaction(session_simulator *session)
 {
+    std::string config = "";
+    std::vector<std::string> options;
+    options.push_back("Prepare timestamp");
+    options.push_back("Run operation");
+    options.push_back("<- go back");
+
+    do {
+        try {
+            print_options(options);
+
+            int choice = choose_num(1, options.size(), "Commit transaction >>");
```

Comment:
Did you try to enter other than numbers when this function is called?

Reply:
Yup, letters, empty string. It works fine.

Reply:
It returns "Choose a number between 1 and 3" here if you enter some other character.

Reply:
In commit_transaction the range is 1 to 4. Hence, it says `Choose a number between 1 and 4`.